pcweb/pages/docs/__init__.py

```diff
@@ -33,11 +33,7 @@ def to_title_case(text: str) -> str:
     title = rx.utils.format.to_snake_case(doc.rsplit("/", 1)[1].replace(".md", ""))
     category = doc.split("/")[-2].title()
     d = flexdown.parse_file(doc)
-    if doc.startswith("docs/library/chakra"):
```

Comment:
I think this needs to be reverted to separate out the different components

Reply:
oh ok, just merged latest radix-docs branch, taking the separation